app/views/partials/_navigation.html.erb

```diff
@@ -25,10 +25,10 @@
 
         <li><%= link_to "My Orders", orders_path(current_user) %></li>
         <li><%= link_to current_user.email, current_user %></li>
-        <li><%= link_to "Sign out", logout_path, method: :delete, class: "btn btn-danger", id: "sign-out" %></li>
+        <li><%= link_to "Sign out", logout_path, method: :delete, id: "sign-out" %></li>
       <% else %>
-        <li><%= link_to "Sign in", login_path %></li>
-        <li><%= link_to "Sign up", new_user_path %></li>
+        <li><%= link_to "Sign in", "#", data: { toggle: "modal", target: "#signInModal" } %></li>
+        <li><%= link_to "Sign up", "#", data: { toggle: "modal", target: "#signUpModal" } %></li>
       <% end %>
```

Comment:
Do we want to make #signInModal a class rather then an ID since we are calling in in multiple places?